src/components/NodeUserInput.tsx

```diff
@@ -111,22 +111,32 @@ export const NodeUserInput = (props) => {
         })
 
       setInputItems([''])
+      setCurrentInputItem('')
       setInputSubmitted(true)
     })
   }
 
   return (
-    <>
-      <h4
-        onClick={() => {
-          setInputIsExpanded(!inputIsExpanded)
-        }}
-        key="userInputHeading"
-      >
-        Send us your input and evidence{' '}
-        {inputIsExpanded ? <FontAwesomeIcon icon={faChevronUp} /> : <FontAwesomeIcon icon={faChevronDown} />}
-      </h4>
-      {inputIsExpanded && (
+    <div css={s.inputContainer}>
```

Comment:
Can you change `forOrAgainst` to a boolean so we don't have to check for string equality in this section?

Reply:
Oh I see, it can have three values. Well I think you can use TypeScript to check that its string values are only 'For' and 'Against'